Samples/iOS-Swift.yml

```diff
@@ -0,0 +1,190 @@
+name: iOS-Swift_xcodegen
```

Comment:
`m`: Why can we not go directly for the `iOS-Swift` but need to append `_xcodegen` which is then removed again in #5118? You are basically making me review unused code